### ISSUE TITLE
Crashes when opening the "progressions" tab

### ISSUE THREAD

**Issue by Simmo86** [Minemaker0430/GDDPIntegration#41]
as the title says when I click into create and then the "progressions" tab it crashes on the loading screen. I've tried fixing it but I'm not sure what else to do

Open this link to see a video of it.

https://github.com/user-attachments/assets/8c93d89a-ae3b-4a44-91db-c06f39912cc1

**And this is the info that it gives me when it does crash if it's helpful at all:**
----------------------------------------------------------------------------
2024-09-16T20:30:19+1000
Whoopsies! An unhandled exception has occurred.

== Geode Information ==
Loader Version: v3.7.1
Loader Commit: d82a344
Bindings Commit: d42aa7e
Installed mods: 6
Problems: 0

== Exception Information ==
Faulty Module: C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\Geode.dll
Faulty Mod: <Unknown>
Exception Code: c0000005 (EXCEPTION_ACCESS_VIOLATION)
Exception Flags: 0
Exception Address: 00007FFA1ABBD877 (Geode.dll + 29d877 (matjson::Value::as_double + 7 | /home/runner/work/geode/geode/cpm-cache/json/7be1271964142fe09a9ce8136cda20080acc1e08/src/value.hpp:58))
Number Parameters: 2
Crashed thread: Main

== Stack Trace ==
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\Geode.dll + 29d877 (matjson::Value::as_double + 7 | /home/runner/work/geode/geode/cpm-cache/json/7be1271964142fe09a9ce8136cda20080acc1e08/src/value.hpp:58)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\geode\unzipped\minemaker0430.gddp_integration\minemaker0430.gddp_integration.dll + 6a0de
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\geode\unzipped\minemaker0430.gddp_integration\minemaker0430.gddp_integration.dll + 68acf
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\geode\unzipped\minemaker0430.gddp_integration\minemaker0430.gddp_integration.dll + 47545
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\geode\unzipped\minemaker0430.gddp_integration\minemaker0430.gddp_integration.dll + 5239a
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\geode\unzipped\minemaker0430.gddp_integration\minemaker0430.gddp_integration.dll + 209f4
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\geode\unzipped\minemaker0430.gddp_integration\minemaker0430.gddp_integration.dll + 1d98c
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\Geode.dll + 7b14b (geode::DefaultEventListenerPool::handle + 491 | /home/runner/work/geode/geode/loader/src/loader/Event.cpp:52)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\Geode.dll + 153fa3 (geode::Task<geode::utils::web::WebResponse,geode::utils::web::WebProgress>::finish::<lambda_1>::operator() + 227 | /home/runner/work/geode/geode/loader/include/Geode/loader/../utils/Task.hpp:275)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\Geode.dll + 8dba2 (geode::Loader::Impl::executeMainThreadQueue + 242 | /home/runner/work/geode/geode/loader/src/loader/LoaderImpl.cpp:879)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\Geode.dll + 67e9d (geode::modifier::AsStaticFunction_update<FunctionQueue,void (FunctionQueue::*)(float)>::Impl<void (FunctionQueue::*)(float)>::function + 29 | /home/runner/work/geode/geode/build/bindings/bindings/Geode/modify/CCScheduler.hpp:115)
 - 0000020043054472 (Hook handler)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\libcocos2d.dll + bbc24 (cocos2d::CCDirector::drawScene + 36)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\libcocos2d.dll + bc33b
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\libcocos2d.dll + 72cd2 (cocos2d::CCApplication::run + 1122)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\GeometryDash.exe + 4a2d69
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\Geode.dll + 28ed4a (gdMainHook + 554 | /home/runner/work/geode/geode/loader/src/platform/windows/main.cpp:181)
 - C:\Users\simeo\OneDrive\Steam\steamapps\common\Geometry Dash\GeometryDash.exe + 4a415a
 - C:\windows\System32\KERNEL32.DLL + 1257d (BaseThreadInitThunk + 29)
 - C:\windows\SYSTEM32\ntdll.dll + 5af28 (RtlUserThreadStart + 40)

== Register States ==
RAX: 80000a00da5b7dea
RBX: 0000000000000000
RCX: 00000200475943a8
RDX: 000002003fdc0000
RBP: 0000000000000000
RSP: 000000569a6ffaf0
RDI: 0000000000000000
RSI: 0000000000000000
RIP: 00007ffa840aaf28
R8:  000002003fc15340
R9:  0000000000000001
R10: 0000000000000003
R11: 000000569a6fda00
R12: 0000000000000000
R13: 0000000000000000
R14: 0000000000000000
R15: 0000000000000000

== Installed Mods ==
x | [v1.9.0] geode.custom-keybinds
x | [v3.7.1] geode.loader
x | [v1.14.0] geode.node-ids
x | [v1.2.4] hiimjustin000.demons_in_between
x | [v1.0.0-beta.42] minemaker0430.gddp_integration
x | [v1.2.0] syzzi.click_between_frames

**Comment by garrdn** [comment]
yeah this also happens to me, ive tried a bunch of things but none of them worked

**Comment by SteveBloX** [comment]
I got almost the same problem

**Comment by SteveBloX** [comment]
I disabled fullscreen and it worked but this still needs to be fixed

**Comment by Minemaker0430** [comment]
https://github.com/Minemaker0430/GDDPIntegration/releases/tag/v1-beta.43 should be fixed here but let me know if it still crashes

**Comment by Simmo86** [comment]
Ok thx it works now

**Comment by Simmo86** [comment]
bruh it worked once and then crashed when i pressed the back button and now its back to the original problem again
@Minemaker0430

**Comment by Minemaker0430** [comment]
update to beta 44, should be on the index

**Comment by Simmo86** [comment]
ok now it works thx :D
